### ISSUE TITLE
Language key for pagination label only for english 

### ISSUE THREAD

**Issue by chmst** [joomla/joomla-cms#44040]
### Steps to reproduce the issue
JLIB_HTML_GOTO_POSITION="Go to %s page" cannot be tranlated properly into German (and probably also other languages).

Any site with a pagination. 
Look into the sourcecode or use WAVE to inspect the site and make ARIA labels visible.

English Pagination texts are correct, or at least I understand.
![grafik](https://github.com/user-attachments/assets/40ebae45-bdbf-43d6-a79f-8f3907236b5f)

 

### Expected result
Normal and correct German translation.  For example "Gehe zur nächsten Seite" and "Gehe zur letzten Seite". 
This is not possible with one language String, 
The translated text is not understandable. 

### Actual result" 
![grafik](https://github.com/user-attachments/assets/9bfafdcf-8bd0-40c5-aaac-717796ab2e30)
It is impossible to translate this into a correct German sentence.

**Comment by brianteeman** [comment]
Not sure I really understand but isnt this a language string issue for the german TT to handle?

The pagination strings are generated from a combination of TWO language strings. For example Next item is a combination of 
 JLIB_HTML_GOTO_POSITION and JNEXT

**Comment by brianteeman** [comment]
JLIB_HTML_GOTO_PAGE="Gehe zur Seite %s"
JLIB_HTML_GOTO_POSITION="Zur Seite %s wechseln"

JLIB_HTML_GOTO_PAGE="Go to page %s"
JLIB_HTML_GOTO_POSITION="Go to %s page"

Seems to me that you are saying the German strings should be
JLIB_HTML_GOTO_PAGE="Gehe zur Seite %s"
JLIB_HTML_GOTO_POSITION="Gehe zur %s Seite"

**Comment by chmst** [comment]
No. German is difficult :) We need extra strings for every case. Will make a PR right now.

**Comment by chmst** [comment]
Please test #44050
